README.md

```diff
@@ -12,23 +12,94 @@ Github CI
 - **kuka_cybertech_support** contains urdf, config and mesh files for KUKA cybertech robots.
 - **kuka_kr_moveit_config** contains configuration files for KUKA KR robots necessary for planning with moveit.
 - **kuka_lbr_iisy_support** contains urdf, config and mesh files for KUKA iisy robots.
-- **kuka_lbr_iisy_moveit_config** contains configuration files for KUKA iisy robots necessary for planning with moveit.
+- **kuka_lbr_iisy_moveit_config** contains configuration files for KUKA LBR iisy robots necessary for planning with moveit.
 - **kuka_agilus_support** contains urdf, config and mesh files for KUKA Agilus robots, it is copied from [kuka_experimental](https://github.com/ros-industrial/kuka_experimental) and ported to ROS2.
-- **kuka_lbr_iiwa_support** contains urdf, config and mesh files for KUKA LBR iiwa 7 robots
+- **kuka_lbr_iiwa_support** contains urdf, config and mesh files for KUKA LBR iiwa robots
+- **kuka_lbr_iiwa_moveit_config** contains configuration files for KUKA LBR iiwa robots necessary for planning with moveit.
 - **kuka_rsi_simulator** implements a basic RSI simulator. It is also copied from [kuka_experimental](https://github.com/ros-industrial/kuka_experimental) and is ported to ROS2. It implements a UDP socket, and receives the commands from an RSI interface and sends back the actual position of the robot (as it is a simulation, it is a simple closed loop, it sends what it received).
 
-## Running the RSI simulator
+## Structure of the support packages
 
+All support packages consist of 4 folders:
+- config: contains joint limits, necessary for time parametrization of trajectories
+- launch: contains launch files to be able to visualize the robot models
+- meshes: contains collision and visual meshes for the robots
+- urdf: contains the xacro files describing the robots, including ros2_control integration (with fake hardware argument)
+
+### Xacro files
+ Each robot has two specific xacro files: a macro ({*robot_name*}_macro.xacro) and another file instantiating this macro ({*robot_name*}.urdf.xacro). Additionally there is a xacro providing ros2_control integration, including the name and type of the hardware interface, hardware parameters and the supported state and command interfaces.
+ Additionally a transmission xacro is provided for gazebo support, but the mechanicalReduction parameters contained within are not valid, only placeholders.
+
+ The macro files contain the links and joints of the main serial chain, including transformations, rotation axes, inertial properties, joint position, velocity and effort limits and the location of the mesh files.
+
+ The macro file follows the ROS-Industrial conventions:
+ - link names are "link_{*i*}"
+ - joint names are "joint_a{*i*}"
```

Comment:
Nit (depending on how you look at this), but `joint_{i}` (in your notation) would be the ROS-Industrial conventions.

The `joint_a{i}` (so with the `a` in there) is a remnant of how joints were named before the current conventions were put in place.

The `a` was really just there to accommodate KUKA users, as that's how joints are named in the KUKA manual (ie: `A1`, `A2`, etc).

I would strongly suggest to convert the `xacro:macro`s to the generic joint names, similar to how manipulators are all called `manipulator`, and `link`s are all just `link_{i}`.

Reply:
Thanks, fixed :)
Do you think anything important is missing from the readme?

Reply:
See https://github.com/ros-industrial/ros_industrial_issues/issues/52 for some discussion about this, which also specifically calls out the situation with the `kuka_experimental` macros.